lib/include/jxl/decode.h

```diff
@@ -130,162 +131,165 @@ typedef enum {
-   * buffer using JxlDecoderSetDCOutBuffer. This occurs if JXL_DEC_DC_IMAGE is
-   * requested and it is possible to decode a DC image from the codestream and
-   * the DC out buffer was not yet set. This event re-occurs for new frames
-   * if there are multiple animation frames.
-   * DEPRECATED: the DC feature in this form will be removed. For progressive
-   * rendering, JxlDecoderFlushImage should be used.
+   * buffer using @ref JxlDecoderSetDCOutBuffer. This occurs if @ref
+   * JXL_DEC_DC_IMAGE is requested and it is possible to decode a DC image from
+   * the codestream and the DC out buffer was not yet set. This event re-occurs
+   * for new frames if there are multiple animation frames. DEPRECATED: the DC
+   * feature in this form will be removed. For progressive rendering, @ref
+   * JxlDecoderFlushImage should be used.
    */
   JXL_DEC_NEED_DC_OUT_BUFFER = 4,
 
   /** The decoder requests an output buffer to store the full resolution image,
-   * which can be set with JxlDecoderSetImageOutBuffer or with
-   * JxlDecoderSetImageOutCallback. This event re-occurs for new frames if there
-   * are multiple animation frames and requires setting an output again.
+   * which can be set with @ref JxlDecoderSetImageOutBuffer or with
+   * @ref JxlDecoderSetImageOutCallback. This event re-occurs for new frames if
+   * there are multiple animation frames and requires setting an output again.
    */
   JXL_DEC_NEED_IMAGE_OUT_BUFFER = 5,
 
   /** The JPEG reconstruction buffer is too small for reconstructed JPEG
-   * codestream to fit. JxlDecoderSetJPEGBuffer must be called again to make
-   * room for remaining bytes. This event may occur multiple times after
-   * JXL_DEC_JPEG_RECONSTRUCTION.
+   * codestream to fit. @ref JxlDecoderSetJPEGBuffer must be called again to
+   * make room for remaining bytes. This event may occur multiple times after
+   * @ref JXL_DEC_JPEG_RECONSTRUCTION.
    */
   JXL_DEC_JPEG_NEED_MORE_OUTPUT = 6,
 
-  /** The box contents output buffer is too small. JxlDecoderSetBoxBuffer must
-   * be called again to make room for remaining bytes. This event may occur
-   * multiple times after JXL_DEC_BOX.
+  /** The box contents output buffer is too small. @ref JxlDecoderSetBoxBuffer
+   * must be called again to make room for remaining bytes. This event may occur
+   * multiple times after @ref JXL_DEC_BOX.
    */
   JXL_DEC_BOX_NEED_MORE_OUTPUT = 7,
 
-  /** Informative event by JxlDecoderProcessInput: basic information such as
-   * image dimensions and extra channels. This event occurs max once per image.
+  /** Informative event by @ref JxlDecoderProcessInput - Basic information such
+   * as image dimensions and extra channels. This event occurs max once per
+   * image.
    */
   JXL_DEC_BASIC_INFO = 0x40,
 
-  /** Informative event by JxlDecoderProcessInput: user extensions of the
+  /** Informative event by @ref JxlDecoderProcessInput - User extensions of the
```

Comment:
Why the change from : to -?

Reply:
Docs fail to build otherwise since it grabs the `:` as part of the function name. I'm not sure how to work around that so I didn't.